namespaces/live.cloud-platform.service.justice.gov.uk/offender-events-prod/resources/topic.tf

```diff
@@ -61,10 +61,8 @@ module "probation_offender_events" {
 
 resource "kubernetes_secret" "prison_data_compliance" {
   metadata {
-    name      = "offender-events-topic-prison-data-compliance"
-    namespace = var.namespace
-    # Remove when namespace has been migrated
-    # namespace = "prison-data-compliance-prod"
+    name      = "offender-events-topic"
```

Comment:
Renamed this as per [current config](https://github.com/ministryofjustice/dps-data-compliance/blob/ad35bcde339c97d38e45b6bea0df8dd9e293f499/helm_deploy/dps-data-compliance/values.yaml#L51)